examples/cloudera-parcel-gcsconnector/create_parcel.sh

```diff
@@ -0,0 +1,134 @@
+#!/bin/bash
+# Copyright 2018 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script creates a Cloudera Parcel which contains Google Cloud Storage connector which
+# enables connections between CDH cluster and Cloud Storage. Using the parcel, CDH can distribute
+# the connector to all the nodes to save time and efforts of distributing and installing the
+# connector.
+
+# User must run the script in a separate folder where only the files to be packaged are present
+# -f: name of the parcel in a single string format without any spaces or special characters.
+# -v: version of the parcel in the format x.x.x (ex: 1.0.0)
+# -o: name of the operating system distribution
+# -d: flag is to be used if you want to deploy the parcel to the cloudera manager parcel repo
+# folder, this flag is optional and if not provided then the parcel file will be created in the
+# same directory where script run.
+
+while getopts f:v:o:d: option
+do
+case "${option}"
```

Comment:
Use indentation of 2 spaces. https://google.github.io/styleguide/shell.xml#Indentation